### PR TITLE
warn users about upgrading to pg15 when using borg

### PR DESCRIPTION
**Pull request by throny**
related: https://github.com/spantaleev/matrix-docker-ansible-deploy/issues/2257

### CODE REVIEW

**Comment by throny** [comment]
I mean, technically, the playbook does not support Borg with postgres15 yet. Personally, I'd have loved to know that before performing the upgrade.
One would have to touch `matrix_backup_borg_supported_postgres_versions`  and add the value `15`, otherwise the supported version check would fail. I will update the text accordingly.

File: docs/maintenance-postgres.md [hunk 1]
```diff
@@ -76,12 +76,17 @@ This is because newer Postgres versions cannot start with data generated by olde
...
 Upgrades must be performed manually.
 
-**Warning: If you're using Borg Backup you probably don't want to upgrade to Postgres 15 yet as there is currently no support.**
-
 This playbook can upgrade your existing Postgres setup with the following command:
 
 	ansible-playbook -i inventory/hosts setup.yml --tags=upgrade-postgres
 
+**Warning: If you're using Borg Backup keep in mind that there is no official Postgres 15 support yet.**
+However, it is possible to use the `latest` or `14` image. Edit `roles/custom/matrix-backup-borg/defaults/main.yml` and adjust the variables accordingly.
```
[the remark this answers is not in the document]
Comment (throny) [reply docs/maintenance-postgres.md]: duh that makes me look stupid..

**Comment by spantaleev** [comment]
Looks good now. Thank you all!

**Comment by throny** [comment]
@etkecc @spantaleev It seems, this is not the full picture here. Just had a look at the output of bormatic:

```
Nov 13 14:55:58 XXXX matrix-backup-borg[501208]: pg_dump: error: server version: 15.0; pg_dump version: 14.5
Nov 13 14:55:58 XXXX matrix-backup-borg[501208]: pg_dump: error: aborting because of server version mismatch
```
Now the guide is misleading again :/ Sorry, should've tested it right away..

**Pull request merged**